Sources/ArcGISToolkit/Components/Offline/PreplannedMapModel.swift

```diff
@@ -211,6 +226,26 @@ extension PreplannedMapModel {
     }
 }
 
+extension PreplannedMapModel.Status: Equatable {
```

Comment:
This is only needed for testing, correct? If so, please move this into the testing target. Not only does it prevent it from being compiled with and included in production code, but it also reduces the likelihood that the API will be abused (since it is a bit of a lie—it doesn't take associated values into account). That might be fine for testing, but not for production code.

Reply:
Yes the equatable conformance is only needed for testing. This conformance can be moved into the testing target but the conformance must be public:
<img width="1198" alt="Screenshot 2024-08-22 at 1 05 47 PM" src="https://github.com/user-attachments/assets/daa66278-2c87-47ba-b53d-9aa494c715c8">

Should this public conformance be added to the testing target? If not, the other options are to add back the `isMatch()` and `assertExpectedValue()` methods for the tests or to move the associated values out of the enum.

Reply:
Yes, just add the `public` access level to make the compiler happy. It shouldn't be necessary and may not be with never versions of Swift, but it doesn't hurt anything.